db-acra-migration/docker-compose.db-acra-migration.yml

```diff
@@ -0,0 +1,166 @@
+version: "3"
+
+services:
+    # Creating keys:
```

Comment:
we don't have acra-writer keys anymore and _storage too as I understand (due to its for acrastructs). let's remove these comments